services/api-server/src/simcore_service_api_server/api/routes/files.py

```diff
@@ -123,11 +123,11 @@ async def upload_file(
     return file_meta
 
 
-# DISABLED @router.post(":upload-multiple", response_model=List[FileMetadata])
-async def upload_files(files: List[UploadFile] = FileParam(...)):
+# DISABLED @router.post(":upload-multiple", response_model=list[FileMetadata])
```

Comment:
Could you add a reason to why this is disabled here?

Reply:
it is actually documented below the function ... will make it more visual